app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+
+{:.note}
+> **Note**: {{site.base_gateway}} provides a `default` Workspace, and similarly {{site.konnect_product_name}} provides a 
+`default` Control Plane. Neither of these can be deleted, so migrating the `default` Workspace to the `default` Control Plane 
+is a logical choice.
+
+Create a new Control Plane for each non-default Workspace:
+
+```txt
+curl -s -H "Authorization: Bearer ${KONNECT_PAT}" \
+  --request POST \
+  --url https://us.api.konghq.com/v2/control-planes \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{"name":"inventory","description":"CP for the Inventory team.","cluster_type":"CLUSTER_TYPE_CONTROL_PLANE"}'
+```
+
+```txt
+curl -s -H "Authorization: Bearer ${KONNECT_PAT}" \
+  --request POST \
+  --url https://us.api.konghq.com/v2/control-planes \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{"name":"sales","description":"CP for the Sales team.","cluster_type":"CLUSTER_TYPE_CONTROL_PLANE"}'
+```
+
+Login into the Konnect UI and validate the new Control Planes.
+
+### Plugins
+
+Konnect supports the majority of plugins available to {{site.base_gateway}}, however,
+{{site.base_gateway}} is ran in Hybrid Mode which limits support for plugins that require direct access
+to a database. 
+
+{:.note}
+> **Note**: Konnect also provides Dedicated Cloud Gateways which 
+further limit plugins that require specialized software agents running on the dataplane hosts. 
+
+In order to migrate plugins from on-premises to Konnect you should review 
+[Konnect Compatibility page](/konnect/compatibility/) for your usage of unsupported
+plugins. Additionally, review if certain configuration values are unsupported which will require additional
+changes to your configuration.
+
+### Custom Plugins
+
+{{site.konnect_product_name}} supports custom plugins with similar restrictions to pre-built plugins. Given 
+that {{site.base_gateway}} runs in a Hybrid deployment mode, custom plugins may not access a database directly
+and can not provide a custom Admin API endpoint. See the Konnect documentation for more details
+on [Custom Plugin support](/konnect/gateway-manager/plugins/#custom-plugins) requirements.
+
+Migrating custom plugins to Konnect requires uploading and associating your custom plugin's `schema.lua` file to 
+the desired Control Plane. This can be done using the 
+[Konnect UI](/konnect/gateway-manager/plugins/add-custom-plugin/) or the 
+[Konnect Control Planes Config API](/konnect/api/control-plane-configuration/latest/#/Custom%20Plugin%20Schemas/list-plugin-schemas).
+
+Just like in on-premises deployments, the custom plugin code must be distributed to the data plane instances.
+
+{:.note}
+> **Note**: Konnect's Dedicated Cloud Gateways can support custom plugins
+but currently require a manual deployment process involving {{site.base_gateway}}'s support personal. Contact your Kong representative
+for more information.
```

Comment:
```suggestion
### Custom plugins

{{site.konnect_short_name}} supports custom plugins with similar restrictions to pre-built plugins. Since {{site.konnect_short_name}} runs in a hybrid deployment mode, custom plugins can't access a database directly
and can't provide custom Admin API endpoints. See the {{site.konnect_short_name}} documentation for more details
on [custom plugin support](/konnect/gateway-manager/plugins/#custom-plugins) requirements.

Migrating custom plugins to {{site.konnect_short_name}} requires uploading and associating your custom plugin's `schema.lua` file to 
the desired control plane. This can be done using the 
[Konnect UI](/konnect/gateway-manager/plugins/add-custom-plugin/) or the 
[Konnect Control Planes Config API](/konnect/api/control-plane-configuration/latest/#/Custom%20Plugin%20Schemas/list-plugin-schemas).

Just like in self-managed deployments, the custom plugin code must be distributed to the data plane instances.

{:.note}
> **Note**: {{site.konnect_short_name}}'s Dedicated Cloud Gateways can support custom plugins
but currently require a manual deployment process involving {{site.base_gateway}}'s support team. 
Contact your Kong representative for more information.
```